examples/OpenTelemetryExamples.cs

```diff
@@ -0,0 +1,49 @@
+﻿using NUnit.Framework;
+using OpenAI.Chat;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+using System;
+using System.Threading.Tasks;
+
+namespace OpenAI.Examples;
+
+public partial class ChatExamples
+{
+    [Test]
+    public async Task OpenTelemetryExamples()
+    {
+        // Let's configure OpenTelemetry to collect OpenAI and HTTP client traces and metrics
+        // and export them to console and also to the local OTLP endpoint.
```

Comment:
nit: Maybe shown in comment how to run a otel collector OR show how to run aspire dashboard?